src/depfile_parser_perftest.cc

```diff
@@ -48,8 +48,8 @@ int main(int argc, char* argv[]) {
       int64_t end = GetTimeMillis();
 
       if (end - start > 100) {
-        int delta = (int)(end - start);
-        float time = delta*1000 / (float)limit;
+        int delta = static_cast<int>(end - start);
+        float time = delta * 1000 / static_cast<float>(limit);
```

Comment:
I believe (but may be mistaken) that it's also necessary to cast delta, and 1000, to float to ensure the compiler doesn't do stupid things.

Reply:
I'm not doubting. The change I suggested should be made regardless of whether the compiler is doing the correct thing. The only doubt is whether the compiler is already doing the correct thing.

Reply:
> The only doubt is whether the compiler is already doing the correct thing.

That's what I meant. Changing the C style casts to static_cast is a complete orthogonal issue.